### PR TITLE
Make Symfony 6.3 the minimum version

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -103,61 +103,61 @@
...
+        "symfony/debug-bundle": "^6.3",
+        "symfony/dependency-injection": "^6.3",
+        "symfony/deprecation-contracts": "^3.0",
+        "symfony/doctrine-bridge": "^6.3",
+        "symfony/doctrine-messenger": "^6.3",
+        "symfony/dom-crawler": "^6.3",
+        "symfony/dotenv": "^6.3",
+        "symfony/error-handler": "^6.3",
+        "symfony/event-dispatcher": "^6.3",
         "symfony/event-dispatcher-contracts": "^2.0 || ^3.0",
```
Comment (ausi) [inline composer.json]:
```suggestion
        "symfony/event-dispatcher-contracts": "^3.0",
```

File: composer.json [hunk 2]
```diff
@@ -103,61 +103,61 @@
...
-        "symfony/framework-bundle": "^6.1.2",
-        "symfony/html-sanitizer": "^6.1",
-        "symfony/http-client": "^5.4 || ^6.0",
+        "symfony/expression-language": "^6.3",
+        "symfony/filesystem": "^6.3",
+        "symfony/finder": "^6.3",
+        "symfony/framework-bundle": "^6.3",
+        "symfony/html-sanitizer": "^6.3",
+        "symfony/http-client": "^6.3",
         "symfony/http-client-contracts": "^2.4 || ^3.1",
```
Comment (ausi) [inline composer.json]:
```suggestion
        "symfony/http-client-contracts": "^3.1",
```

**Review by ausi — DISMISSED** [review]
Didn’t manually test the changes of `ModelArgumentResolver.php`

**Comment by leofeyer** [comment]
As @aschempp pointed out in Slack, this will make it impossible for an extension to be compatible with both Contao 4.13 and 5.3, because Contao 4.13 is only compatible with Symfony 5. There are two possible solutions:

1. Make Contao 4.13 compatible with Symfony 6.
2. Keep Contao 5.3 compatible with Symfony 5.

And I really hope it will not be option 2! 🤞

**Comment by ausi** [comment]
> As @aschempp pointed out in Slack, this will make it impossible for an extension to be compatible with both Contao 4.13 and 5.3, because Contao 4.13 is only compatible with Symfony 5.

I don’t see why that should be the case. The extension simply has to use `"symfony/…": "^5.4 || ^6.4"` that should be straightforward and not a problem at all.

**Review by ausi — APPROVED** [review]

**Pull request merged**